.gitignore

```diff
@@ -3,3 +3,4 @@
 *.exe
 /node_modules/
 /package-lock.json
+.idea
```

Comment:
It's sometimes nice to also ignore in `.dockerignore` so any personal settings doesn't affect the docker build caching.

Reply:
Added the dir too .dockerignore.